demo/lib/orchestrator.go

```diff
@@ -556,9 +556,8 @@ func (e *Orchestrator) StartNode(idxs ...int) {
 
 		fmt.Printf("[+] Attempting to start node %s again ...\n", foundNode.PrivateAddr())
 		foundNode.Start(e.certFolder)
-		trial := 0
 		var started bool
-		for trial < 5 {
+		for trial := 0; trial < 5; trial += 1 {
```

Comment:
This seems to have triggered some extra flakiness in our CI and tests!
Now the "TestLocalOrchestration" test is failing almost always.

We were previously never incrementing the "trial" counter in the loop, so it would keep running non-stop without failing, either until the daemon recovered, or would just "let the other nodes" do their jobs and it used to work thanks to the fact we can support some failure as long as we reached our threshold... 

cc @nikkolasg  @willscott 

😢